### DISCUSSION TITLE
Cookies handling not needed in Python?

### DISCUSSION THREAD

**Discussion by dvermagithub** [timdorr/tesla-api#297] [Q&A]
I was half way though implementing a read, store and re-post of the Cookies in Python per Tim latest authentication instructions.

I came across the fact that requests() in python returns the original cookie automatically as long post is part of the same session.
Code seems to work without requiring storage/return of cookies....maybe not future proof.
Wondering if someone can confirm this about sessions.
TIA

**Comment by MikeBishop** [comment]
Yes, if you first create a requests session and then make all requests on the same session, it handles Cookies for you.  If you just call requests without using a session, each request is stand-alone and you'd have to do Cookies manually.
